content-review-api/public/src/java/org/sakaiproject/contentreview/service/ContentReviewService.java

```diff
@@ -55,6 +55,30 @@ public void queueContent(String userId, String siteId, String taskId, String con
 	throws QueueException;
 	
 	/**
+	 *  Add an item to the Queue for Submission to Turnitin
```

Comment:
The API for content review shouldn't reference turnitin. It should explain in a generic way what this method is for.